### PR TITLE
Fix launch with custom config

### PR DESCRIPTION
**Pull request by bastianhjaeger**

### CODE REVIEW

File: realsense2_camera/config/d455_skynode.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ camera1:
         pointcloud_texture_stream: RS2_STREAM_COLOR
 
         stereo_module:
-          emitter_enabled: true
+          emitter_enabled: 0
```
Comment (markusachtelik) [inline realsense2_camera/config/d455_skynode.yaml:34]: not `false` ?

**Review by markusachtelik — APPROVED** [review]

File: realsense2_camera/launch/rs_launch.py [hunk 2]
```diff
@@ -96,61 +96,38 @@ def generate_launch_description():
...
+    if (LaunchConfiguration('config_file') != ''):
+        realsense_parameters=[set_configurable_parameters(configurable_parameters)
+								             , PythonExpression(["'", config, "/' + ", LaunchConfiguration("config_file")])
+								             ]
+
     if (os.getenv('ROS_DISTRO') == "dashing") or (os.getenv('ROS_DISTRO') == "eloquent"):
         return LaunchDescription(declare_configurable_parameters(configurable_parameters) + [
             # Realsense
             launch_ros.actions.Node(
-                condition=IfCondition(PythonExpression([LaunchConfiguration('config_file'), " == ''"])),
```
Comment (TSC21) [inline realsense2_camera/launch/rs_launch.py:103]: why removing this?

**Review by TSC21 — COMMENTED** [review]
I am ok with these changes but this needs to be also applied in https://github.com/IntelRealSense/realsense-ros/pull/1915. Otherwise we start to accumulate diffs that are not maintainable.

File: realsense2_camera/config/d455_skynode.yaml (hunk 1 above)
Comment (bastianhjaeger) [reply realsense2_camera/config/d455_skynode.yaml:34]: It was complaining about type `bool`. Internally it apparently expects an `int`. 
And for outdoor setups it makes sense to disable it.

File: realsense2_camera/launch/rs_launch.py (hunk 2 above)
Comment (bastianhjaeger) [reply realsense2_camera/launch/rs_launch.py:103]: Due to the modification in 
https://github.com/Auterion/realsense-ros/pull/4/files/6fb4b435a085414fe7030abb2fc01803278abbb5#diff-4f3ab596a35ce7da1b0eb2289ffce9ad499d0970f5d8c98a672f6ddbbc99fa5cR99-R105
this is no longer needed, as `realsense_parameters` handles the difference

**Comment by bastianhjaeger** [comment]
> I am ok with these changes but this needs to be also applied in [IntelRealSense#1915](https://github.com/IntelRealSense/realsense-ros/pull/1915). Otherwise we start to accumulate diffs that are not maintainable.

https://github.com/IntelRealSense/realsense-ros/pull/1942 ok

**Comment by TSC21** [comment]
> > I am ok with these changes but this needs to be also applied in [IntelRealSense#1915](https://github.com/IntelRealSense/realsense-ros/pull/1915). Otherwise we start to accumulate diffs that are not maintainable.
> 
> [IntelRealSense#1942](https://github.com/IntelRealSense/realsense-ros/pull/1942) ok

Great

**Pull request merged**